components/org.wso2.carbon.sp.jobmanager.core/src/main/java/org/wso2/carbon/sp/jobmanager/core/impl/utils/Constants.java

```diff
@@ -35,6 +36,14 @@
     public static final String LONG_TYPE = "longType";
     public static final String FLOAT_TYPE = "floatType";
     public static final String BOOL_TYPE = "booleanType";
+    public static final String ACTIVE = "Active";
```

Comment:
What is `ACTIVE`??
Use meaningful names for constants, prefix them appropriately.